### PR TITLE
As my first contribution here. Added myself to people of Galaxy

### PR DESCRIPTION
**Pull request by EngyNasr**
As a new Galaxy member (finally :)) ), I have added myself to the people.yml of the Freiburg Galaxy Team website. [As described in the SOP]

### CODE REVIEW

File: _data/people.yml [hunk 1]
```diff
@@ -201,7 +201,17 @@ freiburg:
...
-
+            
+    engynasr:
+        name: Engy Nasr
+        title: M.Sc. Inf., Researcher
+        website: http://research.bebatut.fr
+        email: nasr@informatik.uni-freiburg.de
+        location: "Build.: 079, Room: -1005"
+        location_link: "https://goo.gl/maps/zEL3g3KVE8Q2"
+        gitter: engynasr
```
Comment (bgruening) [inline _data/people.yml:212]: please also get an orcid (https://orcid.org) account :)

Have you been able to build this website locally?

Comment (EngyNasr) [reply _data/people.yml:212]: not yet, but I have the way from Helena " If you need to test locally it's a bit complicated, you need to setup a proxy that proxies both your galaxy + the GTN. There's a very short note at the end of https://docs.galaxyproject.org/en/latest/admin/special_topics/gtn.html "

 I will put it on my list to do it

Comment (bgruening) [reply _data/people.yml:212]: ? No this is not needd, please read the README :) 

https://github.com/usegalaxy-eu/website#building

Comment (bgruening) [reply _data/people.yml:212]: Please try to build it locally and get familiar with it. We all add content regular to this website.

Thanks!

Comment (EngyNasr) [reply _data/people.yml:212]: I have successfully built it and currently familiarizing myself :) 

Thank u so much

**Pull request merged**